File: C++/117.cpp
#include <iostream>

using namespace std;

int main()
{
  // 116 - Hola Mundo - Iván - https://github.com/wildfireOfMine
    int bucle = 0;
    string soy = "";
    string nombre = "";
    cin >> bucle;
    for(int i=0; i<bucle; i++) {
        cin >> soy;
        cin >> nombre;
        cout << "Hola, " + nombre + "." << endl;
    }
}

File: Java/117.java
import java.util.Scanner;

/**
 *
 * 117 - La Fiesta Aburrida - Iván - https://github.com/wildfireOfMine
 */

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int bucle = 0;

        bucle = sc.nextInt();
        for (int i = 1; i <= bucle; i++) {
            
            String soy = sc.next();
            String nombre = sc.next();
            System.out.println("Hola, " + nombre + ".");
        }

    }

}
